crates/extrinsics/src/pallet_contracts_primitives.rs

```diff
@@ -78,21 +78,15 @@ pub struct ContractResult<R, Balance, EventRecord> {
     pub debug_message: Vec<u8>,
     /// The execution result of the wasm code.
     pub result: R,
-    /// The events that were emitted during execution. It is an option as event
-    /// collection is optional.
-    pub events: Option<Vec<EventRecord>>,
```

Comment:
What about chains that still have the `events` field?

Reply:
we do not use it in our code, so it is just not parsed from binary data.

Reply:
Where do the events depicted after calling/instantiating/etc. come from if not the `ContractResult`?